index.html

```diff
@@ -157,6 +160,32 @@ <h1 class="title">About me</h1>
                             </div>
                         </div>
                     </section>
+                    <form action="https://formspree.io/f/moqzkbjd" method="post">
+
+                        <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+                        <label for="username"></label>
+                        <input type="text" id="username" name="username" placeholder="username" class="input" required maxlength="30">
+                        <br>
+                        <label for="email"></label>
+                        <input type="email" id="email" name="email" placeholder="email" class="input" required pattern=".*[.com]">
+                        <br><br>
+                        <textarea name="comment" id="comment" cols="30" rows="10" placeholder="write me something..." required maxlength="500"></textarea>
```

Comment:
- [ ] Kindly remove the little black border around the `textarea` to perfectly match the design.

<img width="149" alt="image" src="https://user-images.githubusercontent.com/41163571/229858270-89a67cf6-f845-43f3-aebe-a5a4f3d1cb9d.png">